### ISSUE TITLE
I have noticed two minor problems so far

### ISSUE THREAD

**Issue by Zalnot** [Sunekaer/stoneBlock#719]
I am currently playing on the most recent version 1.0.31 I have not modified the config or the client what so ever and I have more than enough ram dedicated to minecraft and so far in my time playing I have noticed two things. My first problem is I was in the overworld and attempted to use the Ender book from the Cyclic mod to teleport to my storage area. as I teleported I ended up in an area surrounded by lava and I think I was on the surface because I saw clouds and sky. my second problem is with the mechanical users from the Extra Utilities 2 mod  when have more than nine stacks of an item in my inventory and I try to shift double click the item into the inventory into the user it crashes the client.

**Comment by bogointeractive** [comment]
the teleporting to the roof of the world has been there a long time. It was there when I started to play and that was back many versions. Also, that "bug" has been there when you go to the deep dark. That thing is full of nothing like it used to be. I also noticed that the mechanical users work fine for me. I have several of them doing different tasks. So I am not sure what you are doing but it would be nice to see crash logs and or screen shots. We cant help you troubleshoot with nothing to work with.
